operators/nftopology/README.md

```diff
@@ -0,0 +1,152 @@
+# NFTopology Controller
+- Author: @s3wong
+- Status: Work-in-Progress
+- Approver: 
+
+## Description
+NFTopology controller (NTC) takes a network topology centric input and kickstarts the Nephio automation process via generating a PackageVariantSet (PVS) custom resource, and as the associated packages are being deployed, NTC continuously updates statuses to reflect state of deployment of these packages. Primarily, NTC serves as example application to utilize Nephio primitives for network function automations.
+
+![NFTopology High Level Functional Diagram](./img/nftopology-functional.jpg)
+
+NTC contains two controllers, one watches the NFTopology custom resource, and reconcile that intent via constructing a number of PVSs; the other controller watches PackageRevision resources in the management cluster to determine what NF package has been deployed, and updates the NFTopology status accordingly.
+
+## NFTopology CRD
+This is the definition of the NFTopology custom resource:
+
+```go
+// NFInterface defines the specification of network attachment points of a NF
+type NFInterface struct {
+    // Name of the network attachment point
+    Name string `json:"name" yaml:"name"`
+
+    // NetworkInstanceRef is a reference to NetworkInstance. Two NF with attachment to
+    // the same NetworkInstance is considered connected neighbors
+    NetworkInstanceName string `json:"networkInstanceName" yaml:"networkInstanceName"`
+}
+
+// PackageRevisionReference is a temporary replica of PackageRevisionReference used for the
+// ONE Summit
```

Comment:
Perhaps we want to remove this reference to the ONE summit?

Reply:
Removed. Thanks